lib/src/domain/repository/i_repository.dart

```diff
@@ -0,0 +1,3 @@
+abstract class RepositoryInterface {
+
+}
```

Comment:
Remove unused interface

Reply:
Done. Some of the files that are unused, I left them there because I was unsure if I was going to use them in the following modules.